Notice.md

```diff
@@ -6,50 +6,99 @@ the open-source TEE maintained by Linaro, with initial contributions from
 STMicroelectronics, Ericsson and Linaro Limited.
 
 What OP-TEE is
-------
 
-OP-TEE is designed primarily to rely on the ARM TrustZone(R) technology as the
-underlying hardware isolation mechanism. However, it has been structured to be
-compatible with any isolation technology suitable for the TEE concept and goals,
-such as running as a virtual machine or on a dedicated CPU.
+OP-TEE is a Trusted Execution Environment designed as companion to a non secure
```

Comment:
`non-secure` ?

Reply:
I have understood `non secure` is preferred to `non-secure`.
Jens? Jérôme?

Reply:
Correct spelling is *non-secure* according to many sources (such as http://www.dictionary.com/browse/non-secure).